src/test/ui/parser/raw/raw-byte-string-literals.stderr

```diff
@@ -5,10 +5,10 @@ LL |     br"é";
    |        ^
 
 error: found invalid character; only `#` is allowed in raw string delimitation: ~
-  --> $DIR/raw-byte-string-literals.rs:6:6
+  --> $DIR/raw-byte-string-literals.rs:6:5
    |
 LL |     br##~"a"~##;
-   |      ^^^
+   |     ^^^^
```

Comment:
It's be nice if this pointed at the `~` and have the current span be secondary

Reply:
I didn't change that code. I can do that as well.

Reply:
I figured it was preexisting. We can just file a ticket if you prefer not to do it now.

Reply:
I'm on it.

Reply:
I did something, maybe I'm not really happy with it. If you say yes, I'll keep it, if you have concerns, please say them and I will revert that commit.

Reply:
LGTM